packages/webpack-scaffold/index.ts

```diff
@@ -0,0 +1,121 @@
+import jscodeshift from "jscodeshift";
+
+interface IBaseObj {
```

Comment:
I think we should change the name. `IBaseObj` sounds too generic

Reply:
This interface would be scoped to this file so I thought to name it that way. What do you suggest?

Reply:
`IScaffoldBase` or `IScaffoldBaseObject`? Something like this. What do you think?